### DISCUSSION TITLE
Allow VariableMultiValueConverter to be used inside MultiConverter

### DISCUSSION THREAD

**Discussion by Cfun1** [xamarin/XamarinCommunityToolkit#1215] [Ideas]
Moved to https://github.com/xamarin/XamarinCommunityToolkit/issues/1217

Current implementation it's not possible to use a `VariableMultiValueConverter` inside a `MultiConverter`, since the latter requires an object (converter) implementing `IValueConverter` while `VariableMultiValueConverter` does implement `IMultiValueConverter`.
Do you guys think an enhancement could be done to support that ?

```
           <xct:MultiConverter x:Key="MultiConverter">
                <xct:VariableMultiValueConverter .../>
                <xct:BoolToObjectConverter FalseOject=... TrueObject=.../>
            </xct:MultiConverter>
```

**Comment by AndreiMisiukevich** [comment]
Will it be easy to implement?

**Reply by Cfun1** [reply]
@AndreiMisiukevich  At first glance it's not something easy 😅 , you think something worth opening an issue/enhancement ?

**Reply by AndreiMisiukevich** [reply]
If it has a real-life use case, then why not?

**Reply by Cfun1** [reply]
Moved to an issue https://github.com/xamarin/XamarinCommunityToolkit/issues/1217
